package.json

```diff
@@ -21,13 +21,12 @@
   },
   "homepage": "https://github.com/driftyco/ionic-module-template#readme",
   "devDependencies": {
```

Comment:
Well if you have the time, you could check out rangle's doc on it:
https://angular-2-training-book.rangle.io/handout/aot/aot_config.html

This might also be useful:
https://github.com/AngularClass/angular2-webpack-starter/wiki/How-to-include-SCSS-in-components
Not sure how relevant that still is. My understanding is that the whole `ng` + `webpack` is a pretty recent development.